src/api/app/assets/javascripts/webui/requests_table.js

```diff
@@ -1,4 +1,9 @@
 $(document).ready(function() {
+  $('a[data-toggle="tab"]').on('shown.bs.tab', function(){
```

Comment:
Hm on all tabs? This sounds a bit overkill.

Reply:
```$('a[data-toggle="tab"]').length => 4``` sounds ok to me. We don't have other tabs in the view.

Reply:
But this is javascript is not scoped to the page, it's global.

Reply:
oh because the compiled assets right? ok i will scope it better

Reply:
done